src/components/CommentsCard.jsx

```diff
@@ -2,32 +2,38 @@ import axios from "axios";
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import "../assets/CommentsCard.css";
+import Comment from "./Comment";
+import CommentForm from "./CommentForm";
 
 const CommentsCard = () => {
   const { article_id } = useParams();
   const [comments, setComments] = useState([]);
+  const addComment = (text) => {
+    axios.post(
+      `https://nc-news-czlb.onrender.com/api/articles/${article_id}/comments`,
+      { username: "grumpy19", body: text }
+    );
+  };
```

Comment:
This api interaction would be better extracted into it's own file dedicated for api functions, that would clean up the component a bit and help separate your concerns